src/gyro.cpp

```diff
@@ -243,7 +243,7 @@ void gyro::show_params()
  *
  * Computes the distance of a node (x, y) to the Dirichlet boundary, using 
  * tolerance of 1e-10 to define boundary.
- * Highly dependent on dcntl[Param::r0_DB] which is 1e6 is icntl[Param::DirBC_Interior]=0.
+ * Highly dependent on dcntl[Param::r0_DB] which should be very small if icntl[Param::DirBC_Interior]=0.
  * 
  * \param x: the x coordinate of the node
  * \param y: the  coordinate of the node
```

Comment:
in gyro::distBoundary, line 267 maybe one can add that to verbose > 5.

this function is only used in the non-optimized construction of A so also not used that regulary.

Reply:
well spotted!